test/SlowTests/Voron/HugeTransactions.cs

```diff
@@ -0,0 +1,120 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="HugeTransactions.cs" company="Hibernating Rhinos LTD">
+//      Copyright (c) Hibernating Rhinos LTD. All rights reserved.
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.InteropServices;
+using SlowTests.Utils;
+using Voron;
+using Voron.Util.Conversion;
+using Xunit;
+
+namespace SlowTests.Voron
+{
+    public class HugeTransactions : StorageTest
+    {
+        public const long Gb = 1024L * 1024 * 1024;
+        public const long HalfGb = 512L * 1024 * 1024;
+        public const long Mb = 1024L * 1024;
+
+        const string chars = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890 ";
+        public static Random Rand = new Random(123);
+
+        [Theory]
+        [InlineData(1)]
+        [InlineData(2)]
+        [InlineData(4)]
+        [InlineData(6)]
+        [InlineData(9)]
+        public unsafe void CanWriteBigTransactions(long transactionSizeInGb)
```

Comment:
Those are actually running concurrently, and it is really killing the test machine.
We can just have 2 & 6 GB tests, and make sure that they aren't running concurrently.